src/features/plans/components/CategoryCard.tsx

```
@@ -0,0 +1,110 @@
+      <div className="border-t border-border/30 p-4">
+        <PlaceItem place={representativePlace} isRepresentative onSelect={() => {}} />
+
+        <div className="mt-4">
+          <button
+            onClick={e => {
+              e.stopPropagation()
+              onTrigger()
+            }}
+            className="w-full flex items-center justify-center gap-2 rounded-lg bg-primary/10 border border-primary/20 px-4 py-3 text-sm font-medium text-primary hover:bg-primary/20 transition-colors"
+          >
+            <Zap className="h-4 w-4" />
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)